### ISSUE TITLE
Items don't immediately appear in VTW search after approval

### ISSUE THREAD

**Issue by mello99** [VTUL/vtechworks#41]
Hi all, I recently approved an item submitted to VTW; the item was not private, nor did it have an embargo. It's called "Integrating the Adaptive Lighting Database with SHRP 2 Naturalistic Driving Data" (it was submitted to the VTTI community, specifically the NSTSCE collection). Even though I approved it for entry into VTW about 10 minutes ago, I can't find it at all in VTW when I search by title or author. I don't know if VTW indexing is working as planned, but I thought items would be viewable almost immediately after approval - am I wrong? Thanks so much.

**Comment by keithgee** [comment]
@mello99 , VTW indexing was not working as planned. I noticed early this morning that the etds that had loaded yesterday also did not appear in recent submissions.

This was a Solr issue that also occurred once before, soon after we did the install of the new system.
I reindexed this morning and your item appears. Everything is back to normal and newly added items should appear almost immediately.

This problem has happened twice that I know of since we did the install about a month ago. If it continues to be a problem I'll investigate more. However, the daily Tomcat restarts that @kayiwa recommended and ITS has started to implement should resolve the issue within less than 2 days ( < 1 day for Tomcat restart + < 1 day for nighttime index process to run) whenever it does occur.

I'll close this for now and @amandafrench can prioritize it over other things if it continues to happen.
